StripeIdentity/StripeIdentity/Source/Elements/IdentityElementsFactory.swift

```diff
@@ -70,10 +75,17 @@ struct IdentityElementsFactory {
 
     // MARK: DOB
 
-    func makeDateOfBirth() -> DateFieldElement {
-        return DateFieldElement(
-            maximumDate: Date(),
-            locale: locale
+    func makeDateOfBirthSection() -> SectionElement {
+        return  SectionElement(
+            title: String.Localized.date_of_birth,
+            elements: [DateFieldElement(
+                label: "MM / DD / YYYY",
```

Comment:
This should localize, no? For example, in the UK where it would be DD / MM / YYY?